src/clojure_tictactoe/cli/input/input_getter.clj

```diff
@@ -0,0 +1,31 @@
+(ns clojure-tictactoe.cli.input.input-getter
+  (:require [clojure.string :as string]
+            [clojure-tictactoe.game.players :as players]))
+
+(defn get-user-input []
+  (let [user-input (string/trim (read-line))]
+    (if (= user-input "quit")
+      (. System exit 0)
+      user-input)))
+
+(defn continue-to-game []
+  (println "Press enter key to continue")
+  (get-user-input)
+  nil)
+
+(defn parse-move-input
+  ([input]
```

Comment:
You you align these arguments with the method names?

Reply:
I don't understand what you mean by this. @BWoodfork

Reply:
Was supposed to say, "Can you"